src/main/java/co/edu/itc/programacion/biblioteca/repositorio/RepositorioRecursoBase.java

```diff
@@ -0,0 +1,30 @@
+package co.edu.itc.programacion.biblioteca.repositorio;
+
+import java.util.List;
+
+public abstract class RepositorioRecursoBase<T> implements RepositorioRecurso<T> {
+    private List<T> listaRecurso;
+
+    protected  RepositorioRecursoBase() {
+    }
```

Comment:
Instanciar la lista.

```suggestion
protected  RepositorioRecursoBase() {
   this.listaRecurso = new ArrayList();
    }
```